content/en/tracing/advanced_usage/_index.md

```diff
@@ -3,6 +3,131 @@ title: Advanced Languages instrumentation
+
+{{< img src="tracing/enable_trace_search.png" alt="Trace Sampling UI" responsive="true" style="width:100%;">}}
+
+{{< tabs >}}
+{{% tab "Java" %}}
+
+### Automatic Configuration
+
+Trace Search & Analytics can be enabled globally for all web integrations with one configuration parameter in the Tracing Client:
+
+* System Property: `-Ddd.trace.analytics.enabled=true`
+* Environment Variable: `DD_TRACE_ANALYTICS_ENABLED=true`
+
+After enabling, the Trace Search & Analytics UI will now populate, you can get started [here][1].
+
+### Configure Additional Services (optional)
+
+**Configure By Integration**
+
+In addition to setting globally, you can enable or disable Trace Search & Analytics for individual integrations using the following setting:
+
+* System Property: `-Ddd.<integration>.analytics.enabled=true`
+* Environment Variable: `DD_<INTEGRATION>_ANALYTICS_ENABLED=true`
+
+This can be used in addition to the global configuration for any Integrations that submit Custom Services. For example, for JMS spans which comes in as a Custom Service, you can set the following to enable all JMS Tracing in Trace Search & Analytics:
+
+* System Property: `-Ddd.jms.analytics.enabled=true`
+* Environment Variable: `DD_JMS_ANALYTICS_ENABLED=true`
+
+Integration names can be found on the [integrations table](/tracing/languages/java/#integrations).
+
+**Database Services**
+
+Database tracing is not captured by Trace Search & Analytics by default, in order to enable these spans to be collected these can be configured per integration. For example:
+
+* System Property: `-Ddd.jdbc.analytics.enabled=true`
+* Environment Variable: `DD_JDBC_ANALYTICS_ENABLED=true`
+
+**Custom Instrumentation**
+
+Applications with custom instrumentation can enable trace analytics by setting the `ANALYTICS_KEY` tag on the service root span:
+
+```java
+import datadog.trace.api.DDTags;
+import datadog.trace.api.Trace;
+import io.opentracing.Tracer;
+import io.opentracing.util.GlobalTracer;
+
+class MyClass {
+  @Trace
+  void myMethod() {
+    final Span span = GlobalTracer.get().activeSpan();
+    // Span provided by @Trace annotation.
+    if (span != null) {
+      span.setTag(DDTags.SERVICE_NAME, "my-custom-service");
+      span.setTag(DDTags.ANALYTICS_KEY, True);
+    }
+  }
+}
+```
+
+**Configure Sample Rate of APM Events**
+
+When enabling Trace Search & Analytics, the default sample rate is to collect 100% of APM Events. For any services that have been enabled for Trace Search & Analytics in the Tracing Client, you can adjusted the sampling rate for APM Events in your [APM settings][2].
+
+{{< img src="tracing/trace_sampling_ui.png" alt="Trace Sampling UI" responsive="true" style="width:100%;">}}
```

Comment:
This image seems to replicate the intent of the above image.

Reply:
Thanks for pulling that out, that's a mistake. I will adjust before merge.